contracts/multisig/src/msg.rs

```diff
@@ -33,6 +33,14 @@ pub enum ExecuteMsg {
     RegisterPublicKey {
         public_key: PublicKey,
     },
+    AuthorizeCaller {
+        // Adds an address to AUTHORIZED_CALLERS, representing the addresses that can start a signing session.
```

Comment:
move this comment to above the message type `AuthorizeCaller` and don't mention the internal data structure. This is the API, so implementation shouldn't be mentioned here. Say something like "Authorizes a contract to call StartSigningSession". Also, probably add a comment on `StartSigningSession` saying something like "Can only be called by an authorized contract"